src/Nut/DebugEvents.php

```diff
@@ -29,36 +33,39 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $this->displayListeners($output);
-    }
-
-    /**
-     * Output a table of listeners, and their callbacks.
-     *
-     * @param OutputInterface $output
-     */
-    private function displayListeners(OutputInterface $output)
-    {
         $table = new Table($output);
-        $table->setHeaders(array(
-            'Event Name',
-            'Priority',
-            'Listener',
-        ));
+        $rightAligned = new TableStyle();
+        $rightAligned->setPadType(STR_PAD_LEFT);
+        $table->setHeaders([
+            [new TableCell('Event names, target callable listeners, and their priority', ['colspan' => 3])],
+            ['Event Name', 'Callable', 'Priority'],
```

Comment:
I've dropped the (first) title row and changed "Callable" to "Listener" … c.b.f :wink: